### PR TITLE
CF-597: auth-uri is required

### PR DESCRIPTION
**Pull request by shintasmith**
Repose wiki had the wrong sample. I didn't realize auth-uri is a required attribute if ```isAuth``` is set to true. Adding the auth-uri.

### CODE REVIEW

**Comment by rackergs** [comment]
can you bump the version to 1.5.1 in the Modulefile ?

**Comment by shintasmith** [comment]
Done, @gregswift

File: templates/client-auth-n.cfg.xml.erb [hunk 1]
```diff
@@ -24,7 +24,8 @@
 <%- if @token_expire_feed -%>
     <atom-feeds check-interval="<%= @token_expire_feed['interval'] -%>">
         <rs-identity-feed id="identity-token-revocation-feed" uri="<%= @token_expire_feed['feed_url'] -%>" 
-                          isAuthed="true" user="<%= @token_expire_feed['user'] -%>" password="<%= @token_expire_feed['pass'] -%>"/>
+                          isAuthed="true" auth-uri="<%= @token_expire_feed['auth_url'] -%>"
```
Comment (mwhahaha) [inline templates/client-auth-n.cfg.xml.erb]: Since isAuthed is optional, this should be configurable and auth-uri only required when isAuthed=true.

Comment (shintasmith) [inline templates/client-auth-n.cfg.xml.erb]: picky! :-P 

I'll work on it.

**Comment by shintasmith** [comment]
closing this one, will re-submit a new one

**Pull request closed without merging**